docs/contribution_workflow.md

```diff
@@ -65,8 +65,34 @@ fit to add to WinUI, file an issue and start a discussion before proceeding.
 * **DON'T** submit PRs that alter licensing related files or headers. If you 
 believe there's a problem with them, file an issue and we'll be happy to 
 discuss it.
-* **DON'T** add or change public API or UI without filing an issue and discussing with us 
-first: see the [New Feature or API Process](feature_proposal_process.md).
+* **DON'T** add or change public API or UI without filing an issue and 
+discussing it first: see the [New Feature or API Process](feature_proposal_process.md).
+
+## Checks
+
+Each pull request must pass the following checks.
+
+#### WinUI_build_OS
+
+This check essentially makes sure that your change actually builds.
+
+One snag you might hit is a failure in `PeformDEPControlsPort.cmd`. This 
+process validates the compatibility of your change with the port to the Windows
+build system and [Windows.UI.Xaml.Controls](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls).
+Unfortunately this process cannot be run locally without authentication to the 
+Windows build, so if you run into a problem with this validation step you may 
+need the help of a Microsoft employee. You may be able to look at similar code 
+and its use of `BUILD_WINDOWS` to figure out what you need to do, but feel free 
+to @ mention Microsoft team members to ask for help.
+
+#### WinUI-Public-MUX-PR
+
+This check runs automated tests on your change. These tests should match what 
```

Comment:
At the moment, this is just a build validation, it doesn't run tests. Yet.

Reply:
Do we have an issue I can link to for when this will do test verification?